purchase_request_kmitl/wizards/purchase_request_line_make_purchase_order.py

```diff
@@ -15,7 +15,8 @@ def _prepare_purchase_order(self, picking_type, group_id, company, origin):
         vals = super()._prepare_purchase_order(picking_type, group_id, company, origin)
         vals.update({
             "operating_unit_id": self.item_ids.request_id.operating_unit_id.id,
-            "date_range_fy_id": self.item_ids.request_id.date_range_fy_id.id
+            "date_range_fy_id": self.item_ids.request_id.date_range_fy_id.id,
```

Comment:
ต้องเปลี่ยนเป็น account_fiscal_year_id
```suggestion
            "account_fiscal_year_id": self.item_ids.request_id.account_fiscal_year_id.id,
```